package.json

```diff
@@ -22,7 +22,7 @@
     "react-redux": "^7.2.0",
     "react-reveal": "^1.2.2",
     "react-router-dom": "^5.2.0",
-    "react-scripts": "3.4.1",
+    "react-scripts": "^3.4.2",
```

Comment:
Why the change?

Reply:
hmm I think I npm audit'ed because of a high risk warning and this just updated too